icons/yii/yii-plain.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill-rule="evenodd" clip-rule="evenodd" fill="#0073BB" d="M107.131 1.228c-14.253.534-31.578 18.435-25.65 35.13 6.275 16.565 10.296 28.348 11.137 46.83 15.062-15.155 28.901-31.609 30.018-50.7.366-15.493-6.967-26.911-15.505-31.26zM69.549 70.012c-2.139 3.823-4.946 7.978-8.606 12.528-7.691 9.693-15.446 25.593-13.061 44.232 31.771-3.506 41.693-27.353 42.563-40.457-5.987-9.034-16.85-14.596-20.896-16.303zM5.477 12.815c-1.128 14.583 5.11 36.267 30.397 43.169 10.648 3.153 18.109 5.096 26.186 8.028 11.809 4.291 21.078 9.768 28.422 19.221-1.363-16.265-8.062-34.902-18.759-49.441C56.462 13.052 30.034 3.097 5.477 12.815z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M89.023 81.366c-.864-11.402-4.54-18.792-6.331-22.98-1.79-4.189-4.542-8.109-4.544-8.103-.005.018-2.478 13.342-6.923 22.983-.743 1.61-1.728 3.62-2.723 5.238-3.069 5.42-7.521 10.603-10.176 15.857-2.632 5.208-3.12 10.365-2.875 16.229.247 5.896 1.609 11.678 2.916 17.41 4.93-1.063 9.22-2.883 12.932-5.21 9.768-6.124 15.67-15.918 17.348-26.47 0 0 .082-.43.118-.952.757-8.155.506-10.729.258-14.002Z"/><path d="M89.023 81.366c-.864-11.402-4.54-18.792-6.331-22.98-1.79-4.189-4.542-8.109-4.544-8.103v.003-.004l-.657-.976c-14.405-20.221-42.03-30.317-65.371-20.96-1.124 14.174 5.447 38.56 29.42 45.363 9.691 2.981 17.452 2.208 26.963 4.794v.001s9.667 3.37 15.291 8.422c2.53 2.273 5.064 5.263 4.937 8.82.801-8.465.544-11.055.293-14.38Z"/><path fill-rule="evenodd" d="M78.555 38.417c-3.406-9.965-1.962-16.643 4.269-25.935C85.796 8.048 90.928 2.905 95.425 0c18.145 11.37 24.855 32.55 18.182 52.22-4.857 14.312-9.414 20.313-20.935 35.038 1.343-15.77-4.207-26.704-9.657-38.853-1.387-3.09-3.305-6.61-4.46-9.988Z" clip-rule="evenodd"/><path d="M88.731 95.746c.127-3.557-2.406-6.547-4.936-8.82-5.625-5.052-15.292-8.422-15.292-8.422.995-1.618 1.98-3.629 2.722-5.239 4.446-9.64 6.918-22.964 6.923-22.982.002-.006 2.754 3.914 4.544 8.102 1.79 4.189 5.468 11.579 6.332 22.98.251 3.326.508 5.916-.293 14.38Z"/></svg>
```

Comment:
Paths need to be united (there should only be one path for line and plain versions)